lib/led/rgb.js

```diff
@@ -292,21 +293,37 @@ RGB.prototype.color = function(red, green, blue) {
           blue: input.blue
         };
     } else if (typeof input === "string") {
-      // color("#ffffff")
-      if (input.length === 7 && input[0] === "#") {
-        input = input.slice(1);
-      }
 
-      if (!input.match(/^[0-9A-Fa-f]{6}$/)) {
-        throw new Error("Led.RGB.color: invalid color (#" + input + ")");
-      }
+      // color("#ffffff") or color("ffffff")
+      if (input.match(/^#?[0-9A-Fa-f]{6}$/)) {
+
+        // remove the leading # if there is one
+        if (input.length === 7 && input[0] === "#") {
+          input = input.slice(1);
+        }
 
-      // color("ffffff")
-      update = {
-        red: parseInt(input.slice(0, 2), 16),
-        green: parseInt(input.slice(2, 4), 16),
-        blue: parseInt(input.slice(4, 6), 16)
-      };
+         update = {
+          red: parseInt(input.slice(0, 2), 16),
+          green: parseInt(input.slice(2, 4), 16),
+          blue: parseInt(input.slice(4, 6), 16)
+        };
+
+        
+      } else {
+        // color name
+        input = converter.keyword2rgb(input.toLowerCase());
+
```

Comment:
I like it! Was thinking that while copy/pasting but was hesitant to add recursion.  Will do